### ISSUE TITLE
Faster than light 

### ISSUE THREAD

**Issue by mroda88** [GENIE-MC/Generator#120]
While debugging the dark neutrino code we did a check on our units conversion constants. 
One thing we noticed is that the speed of light in our environment can be derived from our units in this way
```C++
double speed_of_light = units::second/units::meter; // this gives us the speed of light in m/s
```
that gives as a result `299802761.341` which is 0.0034 % faster than... you know... light! 
It is a very small discrepancy, but we might want to be aware of this just in case.

**Comment by candreop** [comment]
Correcting erroneous superluminal results became a habbit!

**Comment by mroda88** [comment]
This was fixed with pull request #134
